tensorflow/contrib/learn/python/learn/datasets/__init__.py

```diff
@@ -64,3 +75,33 @@ def load_dataset(name, size='small', test_with_fake_data=False):
     return DATASETS[name](size, test_with_fake_data)
   else:
     return DATASETS[name]()
+
+def make_synthetic(name='circles', n_samples=100, noise=None, *args, **kwargs):
+  """Creates binary synthetic datasets
+
+  Args:
+    name: Name of the dataset to generate (optional, str, default='circles')
```

Comment:
usually we write comments like this name: `str`, name of the dataset to generate.
(the optional and default are easily readable from the function's definition)